common/djangoapps/student/tests/test_views.py

```diff
@@ -723,13 +723,14 @@ def test_resume_course_appears_on_dashboard(self):
         )
 
     @override_waffle_flag(COURSE_UPDATE_WAFFLE_FLAG, True)
-    @override_waffle_flag(CONTENT_TYPE_GATING_FLAG, True)
     def test_content_gating_course_card_changes(self):
         """
         When a course is expired, the links on the course card should be removed.
         Links will be removed from the course title, course image and button (View Course/Resume Course).
         The course card should have an access expired message.
         """
+        config = ContentTypeGatingConfig(enabled=True, enabled_as_of=date(2018, 1, 1))
+        config.save()
```

Comment:
it looks like in some places you use .create and not save and in other places you use save. are they equivalent?

Reply:
Oops. I've been trying to use `.objects.create` everywhere, but I had some early ones that I missed, clearly.